reporters/SponsorReporter.js

```diff
@@ -0,0 +1,49 @@
+const { createFormatFunction, createEtherscanLinkMarkdown } = require("../common/FormattingUtils");
+const chalkPipe = require("chalk-pipe");
+const blue = chalkPipe("blue");
+const bold = chalkPipe("bold");
+
+class SponsorReporter {
+  constructor(expiringMultiPartyClient, walletsToMonitor, priceFeed) {
+    this.empClient = expiringMultiPartyClient;
+    this.walletsToMonitor = walletsToMonitor;
+    this.priceFeed = priceFeed;
+
+    this.web3 = this.empClient.web3;
+
+    this.formatDecimalString = createFormatFunction(this.web3, 2);
+
+    this.collateralCurrencySymbol = "DAI";
```

Comment:
You could grab these from the `empClient.emp` right?

Reply:
right now no. it's actually not sored there, just like in the other bots we actually don't have this info. it's an outstanding Toto to improve.